keadm/cmd/keadm/app/cmd/edge/join_others.go

```diff
@@ -310,6 +333,27 @@ func createV1alpha1EdgeConfigFiles(opt *common.JoinOptions) error {
 	} else {
 		edgeCoreConfig.Modules.EdgeHub.HTTPServer = "https://" + net.JoinHostPort(host, "10002")
 	}
+	if opt.HubType != "" {
```

Comment:
Wouldn't it be more concise to use `case api.ProtocolTypeWS, "": ` directly for the judgment below?

Reply:
This suggestion is very good.